connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -229,8 +235,8 @@ Duration adminTimeout() {
         props.putAll(originalsWithPrefix(SOURCE_CLUSTER_PREFIX));
         props.keySet().retainAll(MirrorClientConfig.CLIENT_CONFIG_DEF.names());
         props.putAll(originalsWithPrefix(CONSUMER_CLIENT_PREFIX));
-        props.put("enable.auto.commit", "false");
-        props.put("auto.offset.reset", "earliest");
+        props.put(ENABLE_AUTO_COMMIT, "false");
+        props.put(AUTH_OFFSET_RESET, CONSUMER_AUTO_OFFSET_RESET);
```

Comment:
If a user sets `somesource->sometarget.consumer.auto.offset.reset=latest` that should be picked up by line 235-237.

So instead of defining a new config on `MirrorConnectorConfig`, could we do something like:
`props.putIfAbsent(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");`

WDYT?

Reply:
ACK